### PR TITLE
Few patches to simplify the recovery process

### PR DESCRIPTION
**Pull request by johnstultz-work**

### CODE REVIEW

**Comment by johnstultz-work** [comment]
Reopening with one more change to the request.

File: README.md [hunk 1]
```diff
@@ -2,221 +2,62 @@ Category
...
 
-To rescue from this error, you need to follow instructions in Above, plug in Jumper J2001 Pin 3-4, using hisi_idt.
+3. Set the board back to normal boot or fastboot mode.
+------------------------------------------------------
 
-Flash Images by Bootloader
-============================
+* Remove power from the board
+* Change Jumper settings, to use normal boot or fastboot mode
+  - find J2001, short Pin 1-2/DIP switch 1 ON
```
Comment (docularxu) [inline README.md]: There is no 'J2001' printed on board. I would like to change the saying.

File: README.md [hunk 2]
```diff
@@ -2,221 +2,62 @@ Category
...
-* [Rescue using hisi_idt](https://github.com/96boards-hikey/tools-images-hikey960/blob/master/README.md#rescue-using-hisi_idt)
-* [Rescue From Endless-Rebooting](https://github.com/96boards-hikey/tools-images-hikey960/blob/master/README.md#rescue-from-endless-rebooting) 
-* [Flash Images by Bootloader](https://github.com/96boards-hikey/tools-images-hikey960/blob/master/README.md#flash-images-by-bootloader)
-* [How to create boot.img and dt.img](https://github.com/96boards-hikey/tools-images-hikey960/blob/master/README.md#how-to-create-bootimg-and-dtimg)
+* [Simple Rescue Flashing](https://github.com/96boards-hikey/tools-images-hikey960/blob/master/README.md#simple-rescue-flashing)
 
 Hardware Versions
 =================
 
 There are two versions of HiKey960: v1 and v2. One important difference you should know is: v1 is using Jumper (J2001), while v2 is using DIP switch (SW2201). Both are located in the bottom side of board.
```
Comment (docularxu) [inline README.md]: SW2201 is what they call it in the schematic. Unfortunately, this is not printed on the real board. So, I think that may mislead some users.

File: README.md [hunk 3]
```diff
@@ -2,221 +2,62 @@ Category
...
 
-* Jumper Pin 1-2 = DIP switch 1
-* Jumper Pin 3-4 = DIP switch 2
-* JUmper Pin 5-6 = DIP switch 3
+* Jumper Pin 1-2 = DIP switch 1  (AutoBoot)
+* Jumper Pin 3-4 = DIP switch 2  (Recovery Mode)
+* JUmper Pin 5-6 = DIP switch 3  (Fastboot Mode)
```
Comment (docularxu) [inline README.md]: Board prints are:
- Auto power
- Boot mode
- Ext boot

These are misleading words, however, we should list them in readme. So end users can easily find them. How about:
* Jumper Pin 1-2 = DIP switch 1  (AutoBoot) (as printed, Auto power)
* Jumper Pin 3-4 = DIP switch 2  (Recovery Mode) (as printed, Boot mode)
* JUmper Pin 5-6 = DIP switch 3  (Fastboot Mode) (as printed, Ext boot)

File: README.md [hunk 4]
```diff
@@ -2,221 +2,62 @@ Category
...
-Start downloading ./sec_fastboot.img@0x1ac00000...
-file total size 3110912
-downlaod address 0x1ac00000
-Finish downloading
-```
 
-3. Burn Images
---------------
+2. Flash the recovery image
+---------------------------
```
Comment (docularxu) [inline README.md]: Add a note:

Note: suppose the device node you get in step one is '/dev/ttyUSB1'

File: README-build-from-source.md [hunk 5]
```diff
@@ -1,194 +0,0 @@
-This document describes how to build ATF, UEFI, Linux kernel and Android for HiKey960 from source.
```
Comment (docularxu) [inline README-build-from-source.md]: Hi, John

I have two questions:
1. UEFI instructions are also required by people who want to have a preview. So I want this information be kept somewhere.
2. Why do you remove linux kernel and Android build instruction?

How about I create a different folder, saying "build-from-source", and put related files "mkdtimg", "mkbootimg", and "README-build-from-source.txt" into it?

Comment (johnstultz-work) [inline README-build-from-source.md]: Why not keep the UEFI instructions with the UEFI source? Right now, nothing in the directory is connected to UEFI. Its all HiSi blobs.

Moving things to a build-from-source directory would be ok, but I think only once the blobs are being used.  Including documentation to build code that isn't being used is just confusing.

File: README.md (hunk 2 above)
Comment (johnstultz-work) [inline README.md]: Not sure what else to refer them to. Having it at least match the schematic is good in my mind. So probably going to leave it here, as there aren't other jumpers/switches on the bottom of the board.

File: README.md (hunk 3 above)
Comment (johnstultz-work) [inline README.md]: Reworked the text here. Thanks for pointing this out!

File: README.md (hunk 4 above)
Comment (johnstultz-work) [inline README.md]: Yea, I've reworked it to be the ttyUSBx to be consistent with the above.

File: README-build-from-source.md (hunk 5 above)
Comment (johnstultz-work) [inline README-build-from-source.md]: One other idea here, is have a UEFI branch, which contains the build from source stuff - maybe as well as current UEFI binaries?  That way folks can tinker with the preview but master is clean?

**Pull request closed without merging**